net/tsaddr/tsaddr.go

```diff
@@ -126,6 +126,18 @@ func Tailscale4To6(ipv4 netaddr.IP) netaddr.IP {
 	return netaddr.IPFrom16(ret)
 }
 
+
+// Tailscale6to4 returns the IPv4 address corresponding to the given
+// tailscale IPv6 address within the 4To6 range. Returns a zero IP
```

Comment:
\n// It returns a

Reply:
But a comma-ok result signature is probably more clear and probably even makes your caller cleaner, and removes an extra IPPrefix.Contains check too!

```go
	if ip4, ok := tsaddr.Tailscale6to4(ip); ok {
		fqdn, code := r.lookupLocalReverse(ip4), name)
```